io/genbank/genbank.go

```diff
@@ -66,14 +68,24 @@ type Meta struct {
 
 // Feature holds the information for a feature in a Genbank file and other annotated sequence files.
 type Feature struct {
-	Type                 string            `json:"type"`
-	Description          string            `json:"description"`
-	Attributes           map[string]string `json:"attributes"`
-	SequenceHash         string            `json:"sequence_hash"`
-	SequenceHashFunction string            `json:"hash_function"`
-	Sequence             string            `json:"sequence"`
-	Location             Location          `json:"location"`
-	ParentSequence       *Genbank          `json:"-"`
+	Type                 string
+	Description          string
+	Attributes           multimap.MultiMap[string, string]
+	SequenceHash         string
+	SequenceHashFunction string
+	Sequence             string
+	Location             Location
+	ParentSequence       *Genbank
+}
+
+type FeatureJSON struct {
+	Type                 string              `json:"type"`
+	Description          string              `json:"description"`
+	Attributes           map[string][]string `json:"attributes"`
+	SequenceHash         string              `json:"sequence_hash"`
+	SequenceHashFunction string              `json:"hash_function"`
+	Sequence             string              `json:"sequence"`
+	Location             Location            `json:"location"`
 }
```

Comment:
Is this just added so that ParentSequence isn't marshalled? Does it increase complexity of remarshalling?

Reply:
This was needed because the multimap library I was using did not play nice with json.marshal, reverted to previous version